Aidan/dockerfileless mode (#232)
latchbio/latch: docs/source/index.md

**Bioinformatics tools face the challenges of irreproducibility.** The lack of proper versioning and dependencies management results in a long tail of poorly documented and unusable bioinformatics software tools.

> The Latch SDK containerizes and versions workflow code in the background each time a workflow is registered to the Latch platform. Container images are constructed by parsing autogenerated or user-defined [Dockerfiles](https://docs.docker.com/engine/reference/builder/). Similarly, versions are user-specified as any unique plaintext string. This allows workflows to be portable and scalable, and for their outputs to be reproducible.

Reviewer: - `The Latch SDK` vs `Latch SDK` is a stylistic choice, but should be kept consistent. most other places drop the `the`
- "Similarly, versions are user-specified as any unique plaintext string." is a stupid sentence but dont care to rewrite it since its old
- rewrote the hypothetical "allows to be" into factual "is"
```suggestion
Latch SDK containerizes and versions workflow code in the background each time a workflow is registered to the Latch platform. Container images are constructed by parsing autogenerated or user-defined [Dockerfiles](https://docs.docker.com/engine/reference/builder/). Similarly, versions are user-specified as any unique plaintext string. The resulting workflows are portable and scalable, and their behavior is reproducible.
```

> Latch SDK containerizes and versions the code in the background each time a workflow is registered to the Latch platform. Container images are constructed by parsing autogenerated or user-defined [Dockerfiles](https://docs.docker.com/engine/reference/builder/). Similarly, versions are user-specified as any unique plaintext string. This behavior is a strict requirement of the toolchain and gives us remarkable guarantees with respect to code reproducibility, portability and scalability.

Reviewer 2: ```suggestion
The Latch SDK containerizes and versions workflow code in the background each time a workflow is registered to the Latch platform. Container images are constructed by parsing autogenerated or user-defined [Dockerfiles](https://docs.docker.com/engine/reference/builder/). Similarly, versions are user-specified as any unique plaintext string. This allows workflows to be portable and scalable, and for their outputs to be reproducible.
```
